src/duo-auth.md

```diff
@@ -110,57 +109,46 @@ The application-specific configuration information retrieved from Duo
 must be added to `guacamole.properties` to describe how Guacamole
 should connect to the Duo service:
 
-duo-api-hostname
+`duo-api-hostname`
 : The hostname of the Duo API endpoint to be used to verify user identities.
-  This will usually be in the form `api-XXXXXXXX.duosecurity.com`, where
-  "XXXXXXXX" is some arbitrary alphanumeric value assigned by Duo. This
-  value will have been generated by Duo when you added Guacamole as an "Web
-  SDK" application, and can be found within the application details in the
-  "API hostname" field. *This value is required.*
-
-duo-integration-key
-: The integration key provided for Guacamole by Duo. This value will
-  have been generated by Duo when you added Guacamole as an "Web SDK"
+  This will usually be in the form {samp}`api-{XXXXXXXX}.duosecurity.com`,
+  where {samp}`{XXXXXXXX}` is some arbitrary alphanumeric value assigned by
+  Duo. This value will have been generated by Duo when you added Guacamole as
+  a "Web SDK" application, and can be found within the application details in
+  the "API hostname" field. *This value is required.*
+
+`duo-client-id`
+: The unique client ID provided for Guacamole by Duo. This value will
+  have been generated by Duo when you added Guacamole as a "Web SDK"
   application, and can be found within the application details in the
-  "Integration key" field. *This value is required and must be EXACTLY
-  20 characters.*
+  "Client ID" field. *This value is required.*
 
-duo-secret-key
-: The secret key provided for Guacamole by Duo. This value will have
-  been generated by Duo when you added Guacamole as an "Web SDK"
+  This value was formerly known as the "integration key" in older versions of
+  Duo's "Web SDK" and was configured with the `duo-integration-key` property
+  in older versions of Guacamole.
+
+`duo-client-secret`
+: The shared secret provided for Guacamole by Duo. This value is will have
```

Comment:
Extra `is` in this line: `This value is will...`

Reply:
Fixed via rebase.